ARExperiment/ARExperiment/UIViewController+Helpers.swift

```diff
@@ -0,0 +1,12 @@
+import UIKit
+
+extension UIViewController {
+    func viewBackgroundColor(to color: UIColor) {
```

Comment:
whilst swift doesn't use `get` in method names, it does unfortunately use `set` :D so this should be:

```suggestion
    func setViewBackgroundColor(to color: UIColor) {
```

the alternative is to make a computed property with a getter and setter, but that would be more that we'd need